definitions/ext-fastly-datacenter/definition.yml

```
@@ -0,0 +1,33 @@
+domain: EXT
+# Derived from the Fastly log ingestion, a datacenter is a Point of Presence
+# https://docs.fastly.com/en/guides/fastly-pop-locations
+type: FASTLY-DATACENTER
```

```suggestion
type: FASTLY_DATACENTER
```

`-` is a reserved character